### PR TITLE
Add dashboard indexes and tables

### PR DESCRIPTION
**Pull request by jarrel-b**
Changes
* Unschedules the migration view because we're done with the migration 🚀 
* Adds indexes to `token_medias`
* Adds validation table for the dashboard

### CODE REVIEW

File: db/migrations/core/000086_unschedule_migration_refresh.up.sql [hunk 1]
```diff
@@ -0,0 +1 @@
+select cron.unschedule((select jobid from cron.job where jobname = 'tokenprocessing-migration-dashboard'));
```
Comment (radazen) [inline db/migrations/core/000086_unschedule_migration_refresh.up.sql:1]: FYI, I think you can unschedule via job name directly, i.e.

`select cron.unschedule('tokenprocessing-migration-dashboard');`

https://github.com/citusdata/pg_cron#what-is-pg_cron

**Review by radazen — APPROVED** [review]
👍👍

File: db/migrations/core/000086_unschedule_migration_refresh.up.sql (hunk 1 above)
Comment (jarrel-b) [reply db/migrations/core/000086_unschedule_migration_refresh.up.sql:1]: ah good to know!

Comment (jarrel-b) [reply db/migrations/core/000086_unschedule_migration_refresh.up.sql:1]: Looks like the first method is idempotent if `tokenprocessing-migration-dashboard` was already unscheduled or doesn't exist.

**Pull request merged**